.github/pull_request_template.md

```diff
@@ -0,0 +1,6 @@
+*Note for BuildXL Team:*
+
+| Command | Description |
+| ------- |-------|
+| `/azp run` | Run all validation|
+| `/azp run Mac Tests PR Validation - GitHub BuildXL.Test` | Runs some mac tests|

```

Comment:
Please don't advertise naked `/azp run`. Since that runs every single optional validation. Please just only list the ones that were 'optional' in our old Azure Repos.